src/pages/index.module.css

```diff
@@ -3,6 +3,11 @@
   width: 100%;
   max-width: 1604px;
   margin: 0 auto;
+
+  /* TODO: remove this min-height,
+  just demo-ing the sticky header in
+  BaseNewLayout */
+  min-height: 150vh;
```

Comment:
Intent is to remove this as mentioned in the TODO, just wanted to demo the `BaseNewLayout` working with longer content (so sticky header behaviour could be confirmed)

Reply:
✅ Removed